pkg/config/user_config.go

```diff
@@ -51,6 +51,7 @@ type GuiConfig struct {
 	CommandLogSize            int                `yaml:"commandLogSize"`
 	SplitDiff                 string             `yaml:"splitDiff"`
 	SkipRewordInEditorWarning bool               `yaml:"skipRewordInEditorWarning"`
+  WindowSize               string              `yaml:"windowSize"`
```

Comment:
This needs to be indented to the right by one more tab

Reply:
Fixed it. Sorry about that